src/main/java/seedu/address/logic/commands/applicant/DeleteApplicantCommand.java

```diff
@@ -41,11 +46,37 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Applicant applicantToDelete = lastShownList.get(targetIndex.getZeroBased());
+        int deleteInterviewCount = deleteApplicantsInterview(model, applicantToDelete);
+
         model.deletePerson(applicantToDelete);
-        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, applicantToDelete),
+        return new CommandResult(
+                String.format(MESSAGE_DELETE_PERSON_SUCCESS, applicantToDelete) + "\n"
+                        + String.format(MESSAGE_DELETE_INTERVIEWS, deleteInterviewCount),
                 getCommandDataType());
     }
 
+    /**
+     * Deletes interviews which are for the applicant to delete.
```

Comment:
Would it be clearer to add in the return value as well? Maybe something like `@return number of interviews deleted`